src/measure_theory/constructions/pi.lean

```diff
@@ -16,6 +16,10 @@ In this file we define and prove properties about finite products of measures
 * `measure_theory.measure.pi`: The product of finitely many σ-finite measures.
   Given `μ : Π i : ι, measure (α i)` for `[fintype ι]` it has type `measure (Π i : ι, α i)`.
 
+To apply Fubini along some subset of the variables, use
+`measure_theory.measure.map_pi_equiv_pi_subtype_prod` to reduce to the situation of a product
+of two measures.
+
```

Comment:
Could you detail this a bit more? For example by adding the equality proven by that lemma.